torchvision/prototype/transforms/_misc.py

```diff
@@ -62,13 +62,14 @@ def __init__(self, transformation_matrix: torch.Tensor, mean_vector: torch.Tenso
         self.transformation_matrix = transformation_matrix
         self.mean_vector = mean_vector
 
-    def _transform(self, inpt: Any, params: Dict[str, Any]) -> Any:
+    def forward(self, *inputs: Any) -> Any:
+        sample = inputs if len(inputs) > 1 else inputs[0]
+        if has_any(sample, PIL.Image.Image):
+            raise TypeError("LinearTransformation does not work on PIL Images")
```

Comment:
This is not going to work. If you want PIL images to be supported you would also need to add them on `_transformed_types`

Reply:
Not sure what you mean. This specifically says that they are not supported. If we remove this error, they will silently pass through.

Reply:
Right now it passes through. He needs to add `_transformed_types = (is_simple_tensor, features.Image, PIL.Image.Image)`

Reply:
@pmeier and @vfdev-5 pointed out to me offline that this is unnecessary because the error is triggered before the use of `_transformed_types`